### PR TITLE
TC_012.005.003, 005, 006| | Footer > Notes > Visibility, clickability, redirection

### PR DESCRIPTION
**Pull request by NKrap**
Added tests for Notes visibility clickability and redirection

https://trello.com/c/LjJnpd6J/420-tc012005003-footer-notes-visibility-and-clickability 
https://trello.com/c/jPZfHFpe/485-tc012005005-footer-notes-clickability?filter=member:devil732704
https://trello.com/c/LlzHLtqd/486-tc012005006-footer-notes-redirection

### CODE REVIEW

File: tests/test_notes_in_footer_is_visible.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+def test_redirect_to_contact_form_from_notes():
+    browser.open(BASE_URL)
+    original_window = browser.driver.current_window_handle
+    s(FooterLocators.NOTES).click()
+    for window_handle in browser.driver.window_handles:
+        if window_handle != original_window:
+            browser.driver.switch_to.window(window_handle)
+            break
+    browser.should(have.url(SOFTWARE_TESTING_BOARD))
+    s("h1.alignwide").should(have.text("Magento 2 Store(Sandbox site) – Notes"))
```
Comment (aqaestuser) [inline tests/test_notes_in_footer_is_visible.py:35]: не должно быть локаторов в тесте
notes.visit()
notes.click_something_that_redirects_somewhere()
somewhere.check_for_redirection()

File: pages/footer.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from selene import browser, by, be, have
+from selene.support.shared.jquery_style import s
+from pages.locators import FooterLocators
+from data.links import BASE_URL
+from selene import command
+
+
+class Footer:
```
Comment (aqaestuser) [inline pages/footer.py:8]: работаем с модулями а не  с классами

File: tests/test_notes_in_footer_is_visible.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+def test_notes_in_footer_is_visible():
+    footer = Footer(browser=browser)
+    footer.open_base_page()
+    footer.scrol_to_footer()
+    footer.move_to_element()
+    footer.is_visible_Notes()
+
+
+def test_notes_in_footer_is_clickable():
```
Comment (aqaestuser) [inline tests/test_notes_in_footer_is_visible.py:17]: def test_notes_in_footer_is_visible(any_page)
    any_page.visit()
    any_page.scoll_to_footer()
    footer.assert_notes_is_visible()

**Review by aqaestuser — APPROVED** [review]
код будет переписан

**Review by MilaArh — APPROVED** [review]
LGTM

**Pull request merged**